PRASCore.jl/src/Simulations/DispatchProblem.jl

```diff
@@ -380,6 +427,58 @@ function update_problem!(
 
     end
 
+
+
+    # Update Demand Response charge/discharge limits and priorities
+    for (i, (borrow_node, borrow_edge, payback_node, payback_edge)) in
+        enumerate(zip(
+        problem.demandresponse_borrow_nodes, problem.demandresponse_borrow_edges,
+        problem.demandresponse_payback_nodes, problem.demandresponse_payback_edges))
+
+        dr_online = state.drs_available[i]
+        dr_energy = state.drs_energy[i]
+        maxenergy = system.demandresponses.energy_capacity[i, t]
+
+        # Update energy payback
+
+        maxpayback = dr_online * system.demandresponses.payback_capacity[i, t]
+        paybackefficiency = system.demandresponses.payback_efficiency[i, t]
+        energy_payback_allowed = dr_energy * paybackefficiency
```

Comment:
Should this be division instead of multiplication? Since an efficiency less than one means you would need to put in more energy to the DR device than the actual energy debt?